data/human/human missions.txt

```diff
@@ -6893,3 +6895,387 @@ mission "Lost Dog: Homeward Bound"
+			`	"Not exactly a rave review, but you're entitled to your opinion," she says, smiling.`
+				goto choice
+
+			label political
+			`	"Oh. Unfortunately for many, everything is political. It was political when the Paradise Worlds tried to block the evacuation. And it was political when the Navy did it anyway without the permission of Parliament. And the same goes for every single act of oppression and liberation since the dawn of humanity. Our oppressors are very political. Maybe you have the hope of being one someday: wealthy, authoritarian, exploitative. Or maybe you just don't care. Either way, you don't sound like a good fit for this project. But thanks for transporting the installations," she says with a smile. She gets her smoothie to-go and walks out.`
+				decline
+
+			label choice
+			`	"I heard from Mariel that you saw me take the photos of the woman and her baby," she says. "I really liked that moment, but not many notice it." She takes a quick glance around before leaning in closer. "We need a captain to transport a team to one planet, quickly land on another, and then pick up the team and return here with everyone safe and sound. I can't tell you much more than that until you accept, but it's nothing too dicey. It's a dangerous universe, though, so who knows?" She looks at you expectantly.`
+			`	"Oh, and we have to leave today. It will pay, of course. <payment>. So, what do you say? Are you in or out?" When she finishes speaking, your order is served while she drinks her smoothie.`
+			choice
+				`	"In."`
+				`	"Out."`
+					goto out
+
+			`	"Great. My name is Alejandra, but call me Ale. Here's the plan: you're going to drop me and a friend off at <planet>, where we'll to film the conditions in a secret Syndicate sweatshop.`
+			`	"Thing is, these aren't just poorly paid factory workers. They're slaves. As you can imagine, we can't get in as regular workers, so with a bit of hacking, we got ourselves jobs on-site as private security. We know that one of the Syndicate's 'trusted contractors' is getting the slaves from the pirates on Covert, and we also know where they're doing the transaction.`
+			`	"So while we get the footage we need in the factories, you'll be going with my associate, Bunker, to spy on the Syndicate's rendezvous with the pirates on <planet>."`
+			choice
+				`	"Hold on, pirates?"`
+				`	"Won't this get me in trouble with the Syndicate?"`
+					goto syndicate
+
+			`	"Don't worry, Bunker is a pro, and we have an array of long-range stealth drones that do the spying. You won't be anywhere near them. You might encounter some lone pirates on the way though, so you'll need to be quick."`
+				goto leave
+
+			label syndicate
+			`	"Well, I did say we had enemies in high places, didn't I? But don't worry. Bunker's a pro. We have an array of long-range stealth drones that do the spying. You won't be anywhere near them, though pirates do frequent the system, so you'll need to be quick."`
+
+			label leave
```

Comment:
I feel like after learning more details about the mission, it'd make sense to provide a second option to say you're in or out.